File: _tmp_svg_colors.py
import re
from collections import Counter
p = r"c:\Dev\01. BD_HOME\assets\BLOCKCHAIN_flow.svg"
with open(p, encoding="utf-8") as f:
    s = f.read()
fills = re.findall(r'fill="(#[0-9A-Fa-f]{3,8})"', s)
strokes = re.findall(r'stroke="(#[0-9A-Fa-f]{3,8})"', s)
print("TOP FILLS")
for k, v in Counter(fills).most_common(30):
    print(v, k)
print("TOP STROKES")
for k, v in Counter(strokes).most_common(30):
    print(v, k)

File: _tmp_svg_colors.mjs
import fs from "fs";
const s = fs.readFileSync("c:/Dev/01. BD_HOME/assets/BLOCKCHAIN_flow.svg", "utf8");
const fills = [...s.matchAll(/fill="(#[0-9A-Fa-f]{3,8})"/gi)].map((m) => m[1]);
const strokes = [...s.matchAll(/stroke="(#[0-9A-Fa-f]{3,8})"/gi)].map((m) => m[1]);
function top(arr) {
  const c = {};
  for (const x of arr) c[x] = (c[x] || 0) + 1;
  return Object.entries(c).sort((a, b) => b[1] - a[1]).slice(0, 35);
}
console.log("FILLS", top(fills));
console.log("STROKES", top(strokes));
